honeybee_energy/properties/model.py

```diff
@@ -27,15 +28,17 @@ class ModelEnergyProperties(object):
     """Energy Properties for Honeybee Model.
 
     Properties:
-        unique_materials
-        unique_constructions
-        unique_face_constructions
-        unique_shade_constructions
-        unique_construction_sets
-        global_construction_set
-        unique_schedule_type_limit
-        unique_schedules
-        unique_shade_schedules
+        * unique_materials
```

Comment:
I know this has been there from the start and I should have commented about this probably sooner but it feels that unique is redundant here as these output must be unique anyways. Also in case we want to keep unique in the name then having it at the end of the name is the recommended approach (e.g. materials_unique). I can't find the link in the documentation right now but I remember part of the logic was better readability in the API. Now you will get `unique_` before what really matters (e.g. materials, construction, etc).

Reply:
Fair point.  Since all of these properties on the model are unique, I think you're right that we can just shorten all of these names.  However, both the `ConstructionSet` and `ProgramType` have separate properties for unique vs. non-unique.  So would you recommend that I change those two cases to follow what you recommend here (that they all have `unique` at the end)?

Reply:
I removed `unique` from all of the model properties.  Thinking it over, you are right that `_unique` makes more sense at the end of the property.  I'll do this for `ProgramType` and `ConstructionSet`.

Reply:
I tend to say yes but it depends on how much work it will be. It looks like a global search and replace problem to me.

Reply:
I just implemented it now and it was not much time.